tutorials/ComplexArithmetic/ComplexArithmetic.ipynb

```diff
@@ -0,0 +1,623 @@
+    "\n",
+    "This mapping allows us to apply complex arithmetic to geometry, and, more importantly, apply geometric concepts to complex numbers. Many properties of complex numbers become easier to understand when viewed through a geometric lens."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Modulus\n",
+    "\n",
+    "One such property is the **modulus** operator. This operator generalizes the **absolute value** operator on real numbers to the complex plane. Just like the absolute value of a number is its distance from $0$, the modulus of a complex number is its distance from $0 + 0i$. Using the distance formula, if $x = a + bi$, then:\n",
+    "\n",
+    "$$|x| = \\sqrt{a^2 + b^2}$$\n",
+    "\n",
+    "There is also a slightly different, but algebraically equivalent definition:\n",
+    "\n",
+    "$$|x| = \\sqrt{x \\cdot \\overline{x}}$$\n",
+    "\n",
+    "Like the conjugate, the modulus distributes over multiplication.\n",
+    "\n",
+    "$$|x \\cdot y| = |x| \\cdot |y|$$\n",
+    "\n",
+    "Unlike the conjugate, however, the modulus doesn't distribute over addition. Instead, the interaction of the two comes from the triangle inequality:\n",
+    "\n",
+    "$$|x + y| \\leq |x| + |y|$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <span style=\"color:blue\">Exercise 6</span>: Modulus.\n",
+    "\n",
+    "**Input:** A complex number $x = a + bi$, represented as a tuple `(a, b)`.\n",
+    "\n",
+    "**Goal:** Return the modulus of this number, $|x|$.\n",
+    "\n",
+    "> You will probably need some mathematical functions to solve the next few tasks. They are available in Python's math library. You can find the full list and detailed information in the [official documentation](https://docs.python.org/3/library/math.html).\n",
+    ">\n",
+    "<br/>\n",
+    "<details>\n",
+    "    <summary><strong>Need a hint? Click here</strong></summary>\n",
+    "    In particular, you might be interested in <a href=https://docs.python.org/3/library/math.html#math.sqrt>Python's square root function</a>\n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@exercise\n",
+    "def modulus(x : Complex) -> float:\n",
+    "    return # ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imaginary Exponents\n",
+    "\n",
+    "The next complex operation we're going to need is exponentiation. Raising an imaginary number to an integer power is a fairly simple task, but raising something to an imaginary power, or raising an imaginary (or complex) number to a real power isn't quite as simple.\n",
+    "\n",
+    "Let's start with raising real numbers to imaginary powers. Specifically, let's start with a rather special real number - Euler's constant, $e$:\n",
+    "\n",
+    "$$e^{ix} = \\cos x + i\\sin x$$\n",
```

Comment:
It might be confusing for the reader to see x here as the angle and then in the next exercise return to having x a complex number... I would suggest using a different name here, to match notation you use later in polar coordinates section:
```suggestion
    "$$e^{i\theta} = \\cos \theta + i\\sin \theta$$\n",
```